src/shared/logger/winstonToolkitLogger.ts

```diff
@@ -3,44 +3,29 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import * as moment from 'moment'
 import * as vscode from 'vscode'
 import * as winston from 'winston'
 import { ConsoleLogTransport } from './consoleLogTransport'
-import { Loggable } from './loggableType'
 import { Logger, LogLevel } from './logger'
 import { OutputChannelTransport } from './outputChannelTransport'
-
-function formatMessage(level: LogLevel, message: Loggable[]): string {
-    // TODO : Look into winston custom formats - https://github.com/winstonjs/winston#creating-custom-formats
-    let final: string = `${makeLogTimestamp()} [${level.toUpperCase()}]:`
-    for (const chunk of message) {
-        if (chunk instanceof Error) {
-            final = `${final} ${chunk.stack}`
-        } else {
-            final = `${final} ${chunk}`
-        }
-    }
-
-    return final
-}
-
-function makeLogTimestamp(): string {
-    return moment().format('YYYY-MM-DD HH:mm:ss')
-}
+import { isError } from 'lodash'
 
 export class WinstonToolkitLogger implements Logger, vscode.Disposable {
-    // forces winston to output only pre-formatted message
-    private static readonly LOG_FORMAT = winston.format.printf(({ message }) => {
-        return message
-    })
-
     private readonly logger: winston.Logger
     private disposed: boolean = false
 
     public constructor(logLevel: LogLevel) {
         this.logger = winston.createLogger({
-            format: winston.format.combine(WinstonToolkitLogger.LOG_FORMAT),
+            format: winston.format.combine(
+                winston.format.splat(),
+                winston.format.timestamp({
+                    format: 'YYYY-MM-DD HH:mm:ss',
```

Comment:
if `2018-10-02T11:47:02.682Z` is the default format why not use it? It is standard and sometimes fractional seconds are useful

Reply:
I'm fine either way. This was the format there previously, so I think I'll leave for now. But we can always switch over.

Here's the space comparison. Latter might be more friendly

```
2020-05-22T21:34:55.820Z [VERBOSE]: Finished loading resource from https://aws-toolkit-endpoints.s3.amazonaws.com/endpoints.json
2020-05-22T21:34:55.824Z [VERBOSE]: Refreshing AWS Explorer due to Region Provider updates
2020-05-22T21:34:56.406Z [VERBOSE]: Getting AccountId from region us-west-2
```

```
2020-05-22 21:34:55 [VERBOSE]: Finished loading resource from https://aws-toolkit-endpoints.s3.amazonaws.com/endpoints.json
2020-05-22 21:34:55 [VERBOSE]: Refreshing AWS Explorer due to Region Provider updates
2020-05-22 21:34:56 [VERBOSE]: Getting AccountId from region us-west-2
```